superset-frontend/src/dashboard/components/FiltersBadge/index.tsx

```diff
@@ -175,10 +175,10 @@ export const FiltersBadge = ({ chartId }: FiltersBadgeProps) => {
       setDashboardIndicators(indicatorsInitialState);
     } else if (prevChartStatus !== 'success') {
       if (
-        chart?.queriesResponse?.[0]?.rejected_filters !==
-          prevChart?.queriesResponse?.[0]?.rejected_filters ||
-        chart?.queriesResponse?.[0]?.applied_filters !==
-          prevChart?.queriesResponse?.[0]?.applied_filters ||
+        chart?.queriesResponse?.rejected_filters !==
+          prevChart?.queriesResponse?.rejected_filters ||
+        chart?.queriesResponse?.applied_filters !==
+          prevChart?.queriesResponse?.applied_filters ||
```

Comment:
Confirmed that the filter's badge disappeared due to this change - queriesResponse is an array so we need that `[0]`. Fix in progress

Reply:
Thank you!